group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -9583,24 +9584,24 @@ public void testHeartbeatDuringRebalanceCausesRebalanceInProgress() throws Excep
     @Test
     public void testListGroups() {
```

Comment:
I don't see any test cases with group types. Should we add some?

Reply:
I think it's there, did you want a different test by any chance?